IBPSA/Fluid/Actuators/Valves/TwoWayButterfly.mo

```diff
@@ -0,0 +1,30 @@
+within IBPSA.Fluid.Actuators.Valves;
+model TwoWayButterfly
+  "Two way valve with the flow characteristic of a typical butterfly valve"
+  extends IBPSA.Fluid.Actuators.Valves.TwoWayPolynomial(
+    final CvData = IBPSA.Fluid.Types.CvTypes.Kv,
```

Comment:
I don't see a reason why setting this final. Can you please provide an assignment for `Cv` and if possible for `Av` based on the diameter?

Reply:
If the user changes `CvData`, `Kv` will get `fixed=false`, but the model also assigns `Kv = 0.0472*(DN*1000)^2 -2.0104*(DN*1000),`. This results in a warning and I don't see another way of working around that. If the warning is ok then we can remove the `final`.

<img width="480" alt="image" src="https://user-images.githubusercontent.com/5970879/102870690-ab25b100-443d-11eb-9426-fcb1d3555840.png">

Reply:
I removed `final`.

Reply:
@Mathadon : I think it is better to use
```modelica
    final CvData = IBPSA.Fluid.Types.CvTypes.Kv,
```
because `Kv` has a binding, which would need to be removed by the user if another option is selected for `CvData`. I will add `final` in my revision.